### ISSUE TITLE
Python crashes when importing brunel into Jupyter

### ISSUE THREAD

**Issue by tutajsen** [Brunel-Visualization/Brunel#226]
jupyter --version
4.2.1

python --version
Python 3.6.0 :: Anaconda 4.3.1 (64-bit)

When running a notebook with 'import brunel' python crashes and the kernel restarts.
Notebook runs fine without the import.

**Comment by danrope** [comment]
Strongly suspect this is due to support we need for JPype in python 3.6.   You may need to use python 3.5 until we can get this resolved.

**Comment by tutajsen** [comment]
Thanks Dan!
I uninstalled Anaconda and tried a bare bones install of python 2.7/3.5 and jupyter using pip/pip3.
Same issue... As soon as I import brunel pyton stops working... Any ideas?

![image](https://cloud.githubusercontent.com/assets/5661541/25006441/e31ae386-202a-11e7-8dc9-4420cef07379.png)

**Comment by Harry-Jia** [comment]
Yes, I met the same issue with Python 3.6 on Win7 64 bit.

**Comment by fstrecker** [comment]
The same for me on a 2.7.6. environment. But Brunel works for other notebooks in my case, when I run the Baseball sample the import works.

**Comment by Harry-Jia** [comment]
I tried the Python 3.5.2 on Win7 64 bit. Same crash issue exists. Is there any switch in python provided to get more debug information while I got such crash?  Thanks a lot.

What I did: 
1. Install Python 3.5.2 (32bit), and install notebook required, and also installed brunel by pip3
2. Try to import brunel in IPython console (import brunel as br), then the Python crashed.  
by the way I can import pandas in the same environment.
